### PR TITLE
Implement update of WFPC2 header for best references in runastrodriz

### PR DESCRIPTION
**Pull request by stsci-hack**
These changes include:

- a new function `wfpc2Data.apply_bestrefs()`
- update to `runastrodriz.process()` to always call `wfpc2Data.apply_bestrefs()` for all WFPC2 input files
- only FLT files will be updated by `runastrodriz.process()` 
- only the distortion reference files will be updated by `runastrodriz.process()`

The new function uses CRDS to update the headers with the best reference files currently available for processing according to CRDS.  If a local CRDS cache is already defined, it will use that, otherwise it will create one in the current working directory then delete it when finished.

The code was tested using data from visit **u2lx05**.

### CODE REVIEW

File: drizzlepac/wfpc2Data.py [hunk 1]
```diff
@@ -429,3 +430,129 @@ def wfpc2_to_flt(imgname):
...
+        Path for the ``CRDS_PATH`` directory on local system.
+        .. warning:: If no value is specified and ``CRDS_PATH`` is not
+        already defined locally, this function will create a temporary
+        ``CRDS_PATH`` directory tree under the directory with the files
+        to be processed, populate it with the latest reference files
+        and mappings needed by CRDS, then delete it when done.
+
+    """
+    starting_dir = os.getcwd()
+    import glob
```
Comment (mdlpstsci) [inline drizzlepac/wfpc2Data.py:475]: This import be at the top of the module.

Comment (stsci-hack) [reply drizzlepac/wfpc2Data.py]: Thanks for catching that!  Removed in 74dd1d7.

**Review by mdlpstsci — DISMISSED** [review]
Code looks fine to me.

**Review by mdlpstsci — APPROVED** [review]
OK.  Looks good.

**Pull request merged**